Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/chimp_upgrade_kit.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  name: C.H.I.M.P. Handcannon Upgrade Kit
+  parent: BaseItem
+  id: WeaponPistolCHIMPUpgradeKit
+  description: An experimental upgrade kit for the C.H.I.M.P.
+  components:
+  - type: Sprite
+    sprite: Objects/Storage/boxes.rsi
+    layers:
+      - state: box_science
+      - state: omegacart   
```

Comment:
this could really do for a unique sprite, since the boxes are used for storage items across the board. Even just a small tweak to the box adding some detail to make it look different from a regular box would be good enough.